ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -175,6 +175,34 @@ router bgp {{ router_bgp.as }}
 {%          endfor %}
 {%       endif %}
 {%     endif %}
+{# address family rt-membership activation #}
+{%     if router_bgp.address_family_rtc is defined and router_bgp.address_family_rtc is not none %}
+   !
+   address-family rt-membership
+{%       if router_bgp.address_family_rtc.peer_groups is defined and router_bgp.address_family_rtc.peer_groups is not none %}
+{%          for peer_group in router_bgp.address_family_rtc.peer_groups | arista.avd.natural_sort %}
+{%             if router_bgp.address_family_rtc.peer_groups[peer_group].activate == true %}
+      neighbor {{ peer_group }} activate
+{%             elif router_bgp.address_family_rtc.peer_groups[peer_group].activate == false %}
+      no neighbor {{ peer_group }} activate
+{%             endif %}
+{%             if router_bgp.address_family_rtc.peer_groups[peer_group].default_route_target is defined %}
+{%                 if router_bgp.address_family_rtc.peer_groups[peer_group].default_route_target.only is defined and router_bgp.address_family_rtc.peer_groups[peer_group].default_route_target.only == true %}
+      neighbor {{ peer_group }} default-route-target only
+{%                 elif router_bgp.address_family_rtc.peer_groups[peer_group].default_route_target.disabled is defined and router_bgp.address_family_rtc.peer_groups[peer_group].default_route_target.disabled is defined == true %}
```

Comment:
You have elif between "disabled" and "only". So that means we will only render one or the other. If that is on purpose, we should make it into a single variable (ex. "mode") that can be set to either "disabled" or "only".
But what is the "disabled" option anyway? I don't see it in the aid.

Reply:
According to TOI the disabled flag is there, but I understand it as only applicable to specific neighbors as opposed to peer-groups.

Reply:
You are right, I changed things to reflect your comments.